ClimsoftVer4/ClimsoftVer4/ucrMetadataPaperArchive.vb

```diff
@@ -4,6 +4,11 @@
             'SetUpTableEntry 
             SetUpTableEntry("paperarchive")
 
+            ucrTextBoxFormId.SetTableNameAndField("obselement", "elementId")
+            ucrTextBoxFormId.PopulateControl()
+            ucrTextBoxFormId.SetValue("elementId")
```

Comment:
@Patowhiz I used **setvalue** on forms where the key control was a textbox and not a combo box. Will this do?

Reply:
What was the intention of this. I can see you are passing elementid to setvalue.

Reply:
I wasn't sure on the text box "equivalent" for the **SetDisplayAndValueMember**.